src/backend/web/static/swagger/api_v3.json

```diff
@@ -4318,6 +4318,61 @@
           }
         ]
       }
+    },
+    "/search_index": {
```

Comment:
What's the reason for not using the typeaheads the same way we use them on web?

Reply:
I had talked about this approach with @fangeugene last week offline and we figured it'd be the best approach.

I am totally fine with using the typeaheads - we'd have to add TypeaheadQuery and other infra to expose it to the API. But this (as in, this PR) approach with utilizing existing Team/Event queries seems quite a bit simpler?

I'm not set on any particular method, this one seems like less code to maintain in the long run I guess? 🤷

Reply:
I have no preference one way or another - just trying to understand the decision!